frontend/webapp/utils/constants/routes.tsx

```diff
@@ -19,8 +19,10 @@ const HOSTNAME = typeof window !== 'undefined' ? window.location.hostname : '';
 const PORT = typeof window !== 'undefined' ? window.location.port : '';
 
 const IS_INGRESSED_DOMAIN = !!HOSTNAME && HOSTNAME !== 'localhost' && PORT === '';
+const IS_DEV = process.env.NODE_ENV === 'development';
+const DEFAULT_BACKEND_HTTP_ORIGIN = 'http://localhost:8085';
 
-const BACKEND_HTTP_ORIGIN = typeof window !== 'undefined' ? (IS_INGRESSED_DOMAIN ? `${PROTOCOL}//${HOSTNAME}` : window.location.origin) : 'http://localhost:3000';
+const BACKEND_HTTP_ORIGIN = IS_DEV || typeof window === 'undefined' ? DEFAULT_BACKEND_HTTP_ORIGIN : IS_INGRESSED_DOMAIN ? `${PROTOCOL}//${HOSTNAME}` : window.location.origin;
```

Comment:
This was later changed to not break current behavior:
```suggestion
const BACKEND_HTTP_ORIGIN = typeof window !== 'undefined' ? (IS_INGRESSED_DOMAIN ? `${PROTOCOL}//${HOSTNAME}` : window.location.origin) : 'http://localhost:3000/';
```